WooCommerce/Classes/ViewRelated/Hub Menu/HubMenu.swift

```diff
@@ -168,7 +168,9 @@ private extension HubMenu {
             case HubMenuViewModel.Customers.id:
                 CustomersListView(viewModel: .init(siteID: viewModel.siteID))
             case HubMenuViewModel.PointOfSaleEntryPoint.id:
-                PointOfSaleEntryPointView()
+                WooCommercePOS.PointOfSaleEntryPointView(hideAppTabBarsCallback: { isHidden in
```

Comment:
I think is good to be explicit here about the view being located in the WooCommercePOS framework, since we're loading this from `WooCommerce` at this point, but happy to iterate further as needed.

Reply:
Yea it's tricky in push navigation when we can't easily hide the tab bar like in UIKit (the design prototype seems to use modal presentation for now but we can update it later). The only other potential workaround I found is [this SO answer](https://stackoverflow.com/a/65058993/9185596), but I haven't tried it and we can revisit the workarounds when we're more sure about the UX.